Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -208,6 +208,47 @@ theorem domRestrict₁₂_apply (f : M →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂]
     (x : p) (y : q) : f.domRestrict₁₂ p q x y = f x y := rfl
 #align linear_map.dom_restrict₁₂_apply LinearMap.domRestrict₁₂_apply
 
+section restrictScalars
+
+variable (R' S' : Type _)
+
+variable [Semiring R'] [Semiring S'] [Module R' M] [Module S' N] [Module R' Pₗ] [Module S' Pₗ]
+
+variable [SMulCommClass S' R' Pₗ] [SMulCommClass S R' Pₗ]
+
+variable [CompatibleSMul M (N →ₗ[S] Pₗ) R' R]
+
+variable [CompatibleSMul N Pₗ S' S]
+
+/-- If `B : M → N → Pₗ` is `R`-`S` bilinear and `R'` and `S'` are compatible scalar multiplications,
+then the restriction of scalars is a `R'`-`S'` bilinear map.
+
+In the case that `R = S` and `R' = S'` the compatibility condition can be easily deduced from
+the scalar tower assumptions
+`[Module R' R] [IsScalarTower R' R M] [IsScalarTower R' R N] [IsScalarTower R' R Pₗ]`. -/
+def restrictScalars₂ (B : M →ₗ[R] N →ₗ[S] Pₗ) : M →ₗ[R'] N →ₗ[S'] Pₗ :=
```

Comment:
I now realize that I PR'd a (less general) version of this already as `LinearMap.restrictScalars₁₂` (so-named to match `LinearMap.compl₁₂`). Can you remove my version in this PR too?